src/fallout.coffee

```diff
@@ -63,15 +60,12 @@
     onFalloutStoreChange: ->
       for name, selector of @_falloutWatchSelectors()
         storeValue = Fallout.store.data.getIn selector
-        unless @state["_#{name}"] is storeValue
+        unless Immutable.is @state["_#{name}"], storeValue
```

Comment:
Yeah, they are immutable values, but from what I understand what I'm doing is just comparing pointers (which should be good enough) and what Immutable.is does is compare all the data. I believe what I had should in theory work, but I haven't done extensive tests. If it's a fallacy then yes, we would need to use `Immutable.is`.